### PR TITLE
*: add pending task metrics in runner and add templ

### PR DESCRIPTION
**Pull request by nolouch**
### What problem does this PR solve?

Issue Number: Ref https://github.com/tikv/pd/issues/7897

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)

image

### Release note

```release-note
None.
```

### CODE REVIEW

File: pkg/ratelimit/runner.go [hunk 1]
```diff
@@ -109,6 +111,9 @@ func (s *ConcurrentRunner) Start() {
 				if len(s.pendingTasks) > 0 {
 					maxDuration = time.Since(s.pendingTasks[0].submittedAt)
 				}
+				for name, cnt := range s.pendingTaskCount {
```
Comment (rleungx) [inline pkg/ratelimit/runner.go:114]: Does it run too frequently?

Comment (nolouch) [reply pkg/ratelimit/runner.go:114]: 5 secs / op

Comment (rleungx) [reply pkg/ratelimit/runner.go:114]: The metrics are collected every 15s.

Comment (nolouch) [reply pkg/ratelimit/runner.go:114]: Yes, The interval should be smaller than the collected interval.

**Review by rleungx — APPROVED** [review]

**Review by HuSharp — APPROVED** [review]

File: metrics/grafana/pd.json [hunk 2]
```diff
@@ -11606,17 +11600,252 @@
...
+          "steppedLine": false,
+          "targets": [
+            {
+              "exemplar": true,
+              "expr": "pd_ratelimit_runner_task_pending_tasks{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\"}",
+              "format": "time_series",
+              "hide": false,
+              "interval": "",
+              "intervalFactor": 2,
+              "legendFormat": "{{task_type}}_({{ runner_name}})",
```
Comment (HuSharp) [inline metrics/grafana/pd.json:11659]:
```suggestion
              "legendFormat": "{{task_type}}_({{runner_name}})",
```

**Pull request merged**